index.html

```diff
@@ -3447,6 +3447,19 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+
+      <section>
+        <h3>Representations of Verifiable Credentials</h3>
+
+        <p>
+Verifiable Credentials can utilize representations other than the standard one defined in this specification.
```

Comment:
Yeah, this was the kind of slippery slope definition of "Verifiable Credential" that I was concerned about... I know others in the group want to call other things "Verifiable Credentials" and some are asserting that we state "W3C Verifiable Credentials"... but this is begging for their to be confusion around the term. Some might argue that this confusion could be created on purpose to confuse the market.

Fundamentally, this comes across as the "JWTs /are/ VCs" rhetoric, and that's problematic. I doubt that is the intent of this PR, but it can certainly be read that way.

We might want to be careful about how we speak to the "compatability" between two media types.

We might want to break this down into multiple categories:

1. Media types can full envelope the Verifiable Credentials data model (e.g. VC-JWT, VC-CWT, that contain an `@context` -- these are `application/vc+ld+json` VCs that are secured using JWT or CWT), and 
2. Media types that express information that can be transformed into the Verifiable Credentials data model (JWTs w/ a default @context, ACDC, Gordian, etc. -- these are not VCs until after they are transformed)

What we want to say is probably more along the lines of these categories being \**compatible*\* with the VCDM (if you can unidirectionally or bidirectionally transform to `application/vc+ld+json`).

Reply:
Fully agreed, @msporny.

(Please go back and code-fence the `@context` in your category 2.)